packages/extensions-sdk/package.json

```diff
@@ -40,31 +40,31 @@
 		"@directus/utils": "workspace:*",
 		"@rollup/plugin-commonjs": "24.1.0",
 		"@rollup/plugin-json": "6.0.0",
-		"@rollup/plugin-node-resolve": "15.0.1",
+		"@rollup/plugin-node-resolve": "15.0.2",
 		"@rollup/plugin-replace": "5.0.2",
 		"@rollup/plugin-terser": "0.4.1",
 		"@rollup/plugin-virtual": "3.0.1",
 		"@vue/compiler-sfc": "3.2.47",
-		"chalk": "4.1.2",
-		"commander": "9.5.0",
-		"esbuild": "0.17.11",
-		"execa": "5.1.1",
-		"fs-extra": "11.1.0",
-		"inquirer": "8.2.5",
-		"ora": "5.4.1",
-		"rollup": "3.19.1",
+		"chalk": "5.2.0",
+		"commander": "10.0.1",
+		"esbuild": "0.17.17",
+		"execa": "7.1.1",
+		"fs-extra": "11.1.1",
+		"inquirer": "9.1.5",
+		"ora": "6.3.0",
+		"rollup": "3.20.6",
 		"rollup-plugin-esbuild": "5.0.0",
 		"rollup-plugin-styles": "4.0.0",
 		"rollup-plugin-vue": "6.0.0"
 	},
 	"devDependencies": {
-		"@directus/tsconfig": "0.0.6",
+		"@directus/tsconfig": "0.0.7",
 		"@types/fs-extra": "11.0.1",
-		"@types/inquirer": "8.2.6",
-		"@vitest/coverage-c8": "0.29.3",
-		"concurrently": "7.6.0",
-		"typescript": "4.9.5",
-		"vitest": "0.29.3"
+		"@types/inquirer": "9.0.3",
+		"@vitest/coverage-c8": "0.30.1",
+		"concurrently": "8.0.1",
```

Comment:
I believe we can remove `concurrently` now 👍

Seems like the script using concurrently was removed in https://github.com/directus/directus/pull/18014/files#diff-bf5b5d2513f9e215c8872acc7ae225035fc81ec09a5cee9ddcd1b8708c52a4f1L44

Adding a screenshot here since that PR is a tad big and the link may not load properly:

![](https://user-images.githubusercontent.com/42867097/233599381-83bb3ab8-59c9-4a91-9dfe-9b421a875335.png)

Reply:
Thanks! 👍